css/main.css

```diff
@@ -0,0 +1,141 @@
+button {
+    border: none;
+    width: 180px;
+    font-weight: 600;
+    font-size: 16px;
+    line-height: 24px;
+    background-color: var(--color-button);
+}
+
+.button-link {
+    color: white;
+}
+
+.hero-section {
+    display: flex;
+    margin: 0px 308px 0px 270px;
+    /* justify-content: center; */
+    align-items: center;
+    padding: 0px 20px;
+    height: 399px;
+}
+
+.hero-title {
+    font-weight: 700;
+    font-size: 44px;
+    line-height: 55px;
+}
+
+.hero-section-list {
+    width: 530px;
+    display: flex;
+    flex-direction: column;
+}
+
+.hero-section p {
+    margin: 20px 0
+}
+
+.posts-section {
+    margin: 60px 270px 40px 270px;
+    background: #EDF7FA;
+
+}
+
+.posts {
+    display: flex;
+    justify-content: space-between;
+    margin: 20px;
+}
+
+.works-link {
+    font-weight: 600;
+    font-size: 16px;
+    line-height: 24px;
+    color: var(--color-view);
+}
+
+.post {
+    display: flex;
+    flex-direction: row;
+}
+
+.posts-article {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 16px;
+    gap: 8px;
+    margin: 20px;
+    width: 438px;
+    height: 226px;
+
+    background: #FFFFFF;
+    border: 2px solid #FFFFFF;
+    border-radius: 8px;
+}
+
+.post-title {
+    font-weight: 600;
+    font-size: 28px;
+    line-height: 42px;
+}
+
+.post-info {
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 150%;
+    color: var(--color-secondary);
+}
+
+.post-text {
+    padding-top: 20px;
```

Comment:
If it's just a space between the title and text, then don't use `padding-top` for it. Padding is an internal space for items, it's used for hovers or backgrounds if we need to extend the interactable area of the item. But if you need just a space under something - always use `margin-bottom` for the top element. Thus you'll always know that spaces between items and elements are handled in the above's element styles.